Doc/library/sqlite3.rst

```diff
@@ -1450,8 +1450,8 @@ This section shows recipes for common adapters and converters.
 
 .. _sqlite3-controlling-transactions:
```

Comment:
I'm not sure if we should change the anchor as well.

Reply:
You could add an additional ref target with the new name, but I would suggest just leaving the old one. The whole idea behind ref targets is that they stay the same even when headings change so you don't need to worry about breaking links.